components/affiliates/integratedBySection.tsx

```diff
@@ -1,49 +1,197 @@
 import React, { FunctionComponent } from "react";
```

Comment:
Instead of duplicating the code, you can reuse components/home/homePartners.tsx (move & rename it to make it more global), (and add the missing partners in it)